javascriptv3/example_code/medical-imaging/README.md

```diff
@@ -13,10 +13,10 @@ _HealthImaging is a HIPAA-eligible service that helps health care providers and
 
 ## ⚠ Important
 
-- Running this code might result in charges to your AWS account.
-- Running the tests might result in charges to your AWS account.
-- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
-- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all) and [Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all).
```

Comment:
My formatter, `prettier`, prefers `-` over `*`. I recommend enabling Git hooks in your environment (.git-hooks/README.md), but that will only catch newly updated files. For now, from the `javascriptv3` directory, you can run `npx prettier example_code/medical-imaging`. That will format all the files in that folder with the defaults.